src/entities/reading_tip.py

```diff
@@ -19,18 +19,10 @@ def format(self, seperator=", ") -> str:
         """Formats the reading tip values to a string using the specified seperator (default ", ").
         """
         values = []
-        if self.id:
-            values.append(f"Id: {self.id}")
-        if self.title:
-            values.append(f"Title: {self.title}")
-        if self.type:
-            values.append(f"Type: {self.type}")
-        if self.author:
-            values.append(f"Author: {self.author}")
-        if self.url:
```

Comment:
nit: Capital Case as in 'URL'. Otherwise LGTM, thank you! A lot of changes on the master branch so rebase is needed.